app/models/persistent_volume_claim.rb

```diff
@@ -6,7 +6,13 @@ class PersistentVolumeClaim < ApplicationRecord
   serialize :requests, Hash
   serialize :limits, Hash
 
+  virtual_column :storage_capacity, :type => :integer
+
   def persistent_volume
     container_volumes.find_by_type('PersistentVolume')
   end
+
+  def storage_capacity
+    capacity[:storage] if capacity
```

Comment:
@yrudman @cben a few questions:
1) Is capacity a serialized hash?  If so, it should be never be nil (no need for `if capacity`)
2) Do we want this method to return nils?  Should it be zero instead?

Reply:
I'm wondering if `capacity.fetch(:storage, 0)` is enough or if we want nils

Reply:
I think nil should mean not set, and 0 - was set to 0

Reply:
Semantically there is difference between 0 and unset capacity.
PVs _probably_ nearly always (?) have a capacity, though there are too many types to be sure...
PVCs don't have capacity when not bound to a PV.  That's not the same as being bound to a 0-size PV (which would be a pretty silly volume).